packages/swing-store/src/transcriptStore.js

```diff
@@ -78,6 +83,35 @@ export function makeTranscriptStore(
     )
   `);
 
+  // The `transcriptCompressedSpans` table holds transcript items for historical
+  // spans in a compressed format.  The shape of this table is a partial
+  // amalgamation of the `transcriptItems` table and the `transcriptSpans`
+  // table.  Whereas the `transcriptItems` table stores one item per row, this
+  // table stores all the items for an entire span in a single compressed blob,
+  // resulting in one row per span just as the `transcriptSpans` table has one
+  // row per span.  The `startPos` and `endPos` fields have the same meanings as
+  // the identically named fields of the `transcriptSpans` table.  However,
+  // these two fields are used slightly differently: since each row captures an
+  // entire completed span, here the `endPos` field will not change over time.
+  // Strictly speaking, the `endPos` field is not actually needed, but we
+  // include it anyway as a redundant sanity check on the state of the record.
+  // Similar reasoning applies to the `incarnation` value, which, while
+  // technically redundant, may prove useful when deleting all the spans in an
+  // incarnation without having to resort to looking at the items table to learn
+  // which spans those are.  This table also lacks the `isCurrent` field on the
+  // grounds that it would be completely useless since in this table it would
+  // always have the value `false`.
+  db.exec(`
+    CREATE TABLE IF NOT EXISTS transcriptCompressedSpans (
+      vatID TEXT,
+      startPos INTEGER, -- inclusive
+      endPos INTEGER, -- exclusive
+      incarnation INTEGER,
+      items BLOB,
```

Comment:
I figured that naming the field `items` (plural) was sufficient, since the corresponding field in the uncompressed table is `item` (singular).  The only thing that renaming it to `compressedItems` buys us is the cue that they're compressed, but the name of the table already tells us that.  There's two things going on here: one is compression and the other is concatenation, both of which interfere with iterating (e.g., in a `for` loop).

How about naming it something like `itemsBlob`, which is still terse but also distinct?